packages/styles/schematics/migrations.json

```diff
@@ -27,10 +27,10 @@
       "description": "Replaces ng-button classes.",
       "factory": "./migrations/ng-buttons/index"
     },
-    "migration-forms": {
+    "migration-form-controls": {
       "version": "5.0.0",
-      "description": "https://getbootstrap.com/docs/5.0/migration/#forms",
-      "factory": "./migrations/forms/index"
+      "description": "Applies automatic migrations on form-control elements.",
+      "factory": "./migrations/form-controls/index"
```

Comment:
Is this change necessary? "form-controls" only refers to inputs, but migration also changes other types of form elements

Reply:
For me form-controls refers to all form-controls (input, select, textarea, checkbox, radio, etc.) but I can see your point and will rename it.